### PR TITLE
Made Allocation options dropdown responsive

### PR DESCRIPTION
**Pull request by alexvlsem**
[DE46880](https://rally1.rallydev.com/#/?detail=/defect/622938061589&fdp=true): Allocation options dropdown is not responsive
Removed 1st column sticky behavior of the mapping table when the screen is narrower than 768px 
Added an entry to the activities serge file for the d2l-activity-evaluation-editor

### CODE REVIEW

File: activities.serge.json [hunk 1]
```diff
@@ -274,5 +274,28 @@
...
       "tr-tr tr"
     ]
+  },
+  {
+    "name": "activityEvaluationEditor",
+    "parser_plugin": {
+      "plugin": "parse_js"
+    },
+    "source_match": "en\\.js$",
+    "source_dir": "components/d2l-activity-evaluation-editor/lang",
```
Comment (sarahboyd) [inline activities.serge.json:284]: Does this need to have the `d2l-activity-editor` as well?

```suggestion
    "source_dir": "components/d2l-activity-editor/d2l-activity-evaluation-editor/lang",
```

File: activities.serge.json [hunk 2]
```diff
@@ -274,5 +274,28 @@
...
+    "output_file_path": "components/d2l-activity-evaluation-editor/lang/%LANG%.js",
```
Comment (sarahboyd) [inline activities.serge.json:285]: Same as above

```suggestion
    "output_file_path": "components/d2l-activity-editor/d2l-activity-evaluation-editor/lang/%LANG%.js",
```

File: activities.serge.json (hunk 1 above)
Comment (alexvlsem) [reply activities.serge.json]: Good catch. Thank you!

File: activities.serge.json (hunk 2 above)
Comment (alexvlsem) [reply activities.serge.json]: Updated

**Pull request merged**